.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -22,17 +22,18 @@ Steps to reproduce the behavior:
 ## CodePen repro example
 
 Please provide a reproduction of the bug in a codepen, if possible. Here’s how
-
+<!--
```

Comment:
Did you mean this to be 1 line further up to include the "Please provide..." text?

Reply:
Updated. Missed that line. 😸